Automattic/wp-calypso
# Stats: fix search and stats in my plan

Related to p1722531890323179-slack-C82FZ5T4G

## Proposed Changes

* Added missing product slug support for Stats and Search


## Testing Instructions


* Switch to user per slack convo: p1722531890323179-slack-C82FZ5T4G
* Ensure no icon/description missing

| Before | After |
|--------|------|
|![image](https://github.com/user-attachments/assets/6322b713-b2ba-4e1c-bfac-397e64a7940c) |<img width="884" alt="image" src="https://github.com/user-attachments/assets/6fadeed2-7d8b-4cf4-8b59-d7e0f3bb242b"> |

## Pre-merge Checklist
